blog/_posts/2020-10-02-false-positives.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Google warns about root.cern"
+layout: archive
+author: Axel Naumann
+---
+
+Thank you for your reports of Google warning about some pages on https://root.cern. Let me give you some background:
+
+We have investigated virus scanner reports on some of ROOT's binaries. The ROOT team has invested about 2.5 work days into this investigation: we took this seriously. On top of that, CERN IT's security team has investigated this independently, and also invested a notable amount of working hours. We were able to create binaries that were diagnosed as infected, by building `hadd.exe` "from scratch". We are convinced that these reports are false positives. We found out that upgrading the compiler works around the issue. 
+
+We could report this as false positive to the vendors whose virus scanner was reporting some of ROOT's binaries. This has a high latency, especially when adding the latency between the virus scanner engines tweaking their patterns and Google removing root.cern from the list of flagged sites.
+
+We could argue with Google. This does not seem like a fast track option either.
+
+Instead, we have removed the file in question. We are upgrading the compiler on our build machines, which means that ROOT versions up to 6.22/02 cannot be released anymore, and which required backporting several changes from master to v6-22-00-patches.
```

Comment:
I don't understand "ROOT versions up to 6.22/02 cannot be released anymore", can you clarify in the text what this means for users?